desktop-src/search/-search-sql-literalvaluecomparison.md

```diff
@@ -56,6 +56,8 @@ In conjunction with the "=" operator, Windows Search Structured Query Language (
 ...WHERE <column> <comparison operator> [BEFORE | AFTER](<https://msdn.microsoft.com/library/Ff637626(v=MSDN.10).aspx>)
 ```
 
+Note: to compare dates, you must use [DATEADD](https://learn.microsoft.com/en-us/windows/win32/search/-search-sql-dateadd).
```

Comment:
Nit: should we move this earlier? For example, after:

> The literal can be any string, numeric, hexadecimal, Boolean, or date literal, enclosed in single quotation marks. Only exact matches are recognized, and wildcard characters are ignored. The literal can also be cast to another type.

Reply:
I think it makes sense to keep it in the Comparison section since it is specifically about comparing date literals.